### PR TITLE
Add integration tests and use github actions

### PR DESCRIPTION
**Pull request by fivethreeo**
+ webpack5 support

### CODE REVIEW

**Comment by fivethreeo** [comment]
Also adds support for webpack5

**Comment by wmertens** [comment]
:-/ sadly, the most recent version is on the use-the-fork branch. Could you check if it works for you with  `start-server-webpack-plugin@beta`?

The mechanism changed a lot, so I'm afraid you'll need to do some adjustments to make it work on the new branch.

File: src/StartServerPlugin.js [hunk 1]
```diff
@@ -137,7 +139,7 @@ export default class StartServerPlugin {
     };
 
     if (inspectPort) {
-      clusterOptions.inspectPort = inspectPort;
+      clustertOptions.inspectPort = inspectPort;
```
Comment (wmertens) [inline src/StartServerPlugin.js]: typo

File: test.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+yarn run test & \
```
Comment (wmertens) [inline test.sh]: the `\` and next indentation isn't needed here, the next lines are their own command

File: .github/workflows/nodejs.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        node-version: [10.x, 12.x]
+        webpack-version: [webpack@latest, webpack@beta]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: NODE_ENV=development yarn install
```
Comment (wmertens) [inline .github/workflows/nodejs.yml]: I'd rather limit the project to npm - it's gotten reasonably fast lately

**Review by wmertens — CHANGES_REQUESTED** [review]
Can you remove the yarn dependency, and rebase this on use-the-fork? Good stuff in any case!

**Comment by fivethreeo** [comment]
Fixed the issues, but the travis build does not work. Maybe some caching?

**Comment by fivethreeo** [comment]
Working travis build now

**Pull request closed without merging**